legal-api/src/legal_api/models/db.py

```diff
@@ -15,13 +15,279 @@
 
 These will get initialized by the application using the models
 """
-from flask_sqlalchemy import SQLAlchemy
+from datetime import datetime
+
+from flask import current_app
+from flask_sqlalchemy import SignallingSession, SQLAlchemy
+from sql_versioning import TransactionManager, debug
+from sql_versioning import disable_versioning as _new_disable_versioning
+from sql_versioning import enable_versioning as _new_enable_versioning
+from sql_versioning import version_class as _new_version_class
+from sqlalchemy import event, orm
+from sqlalchemy.orm import Session, mapper
 from sqlalchemy_continuum import make_versioned
+from sqlalchemy_continuum import version_class as _old_version_class
+from sqlalchemy_continuum.manager import VersioningManager
 
 
 # by convention in the Flask community these are lower case,
 # whereas pylint wants them upper case
 db = SQLAlchemy()  # pylint: disable=invalid-name
 
-# make_versioned(user_cls=None, plugins=[FlaskPlugin()])
-make_versioned(user_cls=None)
+
+class Transaction(db.Model):
+    """This class manages the transaction."""
+
+    __tablename__ = 'transaction'
+
+    id = db.Column(
+                db.BigInteger,
+                db.Sequence('transaction_id_seq'),
+                primary_key=True,
+                autoincrement=True
+            )
+    remote_addr = db.Column(db.String(50), nullable=True)
+    issued_at = db.Column(db.DateTime, default=datetime.utcnow, nullable=True)
```

Comment:
I don't think this should be nullable?

Reply:
You mean the `issued_at`? Don't see any issues for now, but I just copied what `continuum` is doing now that it's nullable by default, but I set it `True` explicitly.

Reply:
ok, maybe leave it as `nullable=True` then.